app/src/main/java/org/mozilla/fenix/tabstray/NavigationInteractor.kt

```diff
@@ -171,12 +172,12 @@ class DefaultNavigationInteractor(
 
     override fun onSaveToCollections(tabs: Collection<Tab>) {
         metrics.track(Event.TabsTraySaveToCollectionPressed)
+        tabsTrayStore.dispatch(TabsTrayAction.ExitSelectMode)
```

Comment:
Found that this is causing the race between showing the floating new tab button and showing the snackbar.  Moving this so we exit the select mode when the dialog is shown removes the possibility of that race.

This doesn't change the behaviour for both positive or negative case.

Reply:
Thank you for figuring this out! Sorry it turned out to be unrelated.